src/Search.php

```diff
@@ -109,7 +109,8 @@ class Search implements Builder, Pipeable
     /** @var null|Highlight */
     private $highlight;
 
-    //slop
+    /** @var int */
+    private $slop = 0;
```

Comment:
Initial (default) value should be `null` otherwise we will always have the `SLOP` parameter in the query (and most of times with the value set to `0`)